packages/block-library/src/template-part/edit/index.js

```diff
@@ -99,7 +104,7 @@ export default function TemplatePartEdit( {
 			return {
 				innerBlocks: getBlocks( clientId ),
 				isResolved: hasResolvedEntity,
-				isMissing: hasResolvedEntity && ! entityRecord,
+				isMissing: hasResolvedEntity && isEmpty( entityRecord ),
```

Comment:
Does this always return an empty object? In general could we adjust the check without adding the lodash dependency?

Reply:
The `getEditedEntityRecord` will return an object. Empty if no edited entity was found.

https://github.com/WordPress/gutenberg/blob/22814c75b0c0041493f791399c70299a037bf3f3/packages/core-data/src/selectors.js#L491-L495

I thought about using `Object.entries(entityRecord).length === 0`, but `isEmpty` is more clear IMO, and we're already using lodash in other blocks.

Reply:
In general there were discussion and and an effort to loose lodash dependencies. It's not a big deal for sure, but one could argue that since we know that it returns an `object`, a specific js check like `Object.keys` makes it also clear about what  type we are expecting.

Reply:
Thanks, Nik, and it makes sense if we're trying to lose `lodash` dep. I will keep it in mind for the future.